backend/business/core/collectors/github/fetchers/repos_info_fetcher.go

```diff
@@ -119,3 +134,25 @@ func (f *ReposInfoFetcher) formatCSV(data []repoInfo) (string, error) {
 	}
 	return csvString, nil
 }
+
+func (f *ReposInfoFetcher) buildJSON(data []repoInfo) *map[string]RepoInfoResult {
```

Comment:
Nit: matches the func definition in the other fetchers
```suggestion
func (f *ReposInfoFetcher) buildResult(data []repoInfo) *map[string]RepoInfoResult {
```